### DISCUSSION TITLE
[Proposal] Inline try statement

### DISCUSSION THREAD

**Discussion by matheus-kirchesch** [dotnet/csharplang#7677] [Language Ideas]
I want to propose a simple feature that is similar to the inline using statement.

This:
```
HttpResponseMessage response;
try
{
  response = await httpClient.SendAsync(requestMessage);
}
catch (Exception ex)
{
  Console.WriteLine($"I posted this by accident trying to skip a line...");
  break;
}
```

Would become this:
```
try HttpResponseMessage response = await httpClient.SendAsync(requestMessage);
catch (Exception ex)
{
  Console.WriteLine($"..");
  break;
}
```

**Comment by HaloFour** [comment]
See: #1015 and #4676

**Comment by matheus-kirchesch** [comment]
> See: #1015 and #4676

Same idea, I still think it is a good idea to highlight the inline declaration of the variable to be used outside of the try scope (that doesn't exist anymore), the same way we can do with using.
